Taarafo.Portal.Web/Brokers/Apis/ApiBroker.Groups.cs

```diff
@@ -0,0 +1,18 @@
+// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Taarafo.Portal.Web.Models.Groups;
+
+namespace Taarafo.Portal.Web.Brokers.Apis
+{
+    public partial class ApiBroker
+    {
+        private const string GroupsRelativeUrl = "api/groups";
+        public async ValueTask<List<Group>> GetAllGroupsAsync() =>
```

Comment:
New line break required

Reply:
I added a line break in the code rub titled "CODE RUB: Added Line Break."